exercises/rotational-cipher/cannonical-data.json

```diff
@@ -0,0 +1,72 @@
+{
+    "#": [
+        "The tests are a series of rotation tests and a bonus test: ",
+        "* Rotational tests",
+        "* non-ASCII downgrade tests (bonus)"
+    ],
+    "rotate": {
+        "description": ["Test rotation from English to ROTn"],
+        "cases": [
+            {
+                "description": "rotate a by 1",
+                "text": "a",
+                "shift_key": 1,
+                "expected": "b"
+            },
+            {
+                "description": "rotate a by 26, same output as input",
+                "text": "a",
+                "shift_key": 26,
+                "expected": "a"
+            },
+            {
+                "description": "rotate a by 0 same input as output",
+                "text": "a",
+                "shift_key": 26,
+                "expected": "a"
+            },
+            {
+                "description": "rotate m by 13",
+                "text": "m",
+                "shift_key": 13,
+                "expected": "z"
+            },
+            {
+                "description": "rotate n by 13 first wrap around alphabet",
```

Comment:
I would get rid of the word "first" as it's not clear whether it means it's the first case that does so (which is what I meant... unless you count 26 as wrapping around all the way), or the first shift_key that would, or the first letter that would (both of which depend)